sonar-dotnet-shared-library/src/test/java/org/sonarsource/dotnet/shared/plugins/ProjectTypeCollectorTest.java

```diff
@@ -35,48 +35,22 @@ public void withNoProjects() {
   public void withNoFiles_skip() {
     ProjectTypeCollector underTest = new ProjectTypeCollector();
 
-    // this will be skipped, considered as top-level module
-    addProjectWithNoFiles(underTest);
```

Comment:
After this change both `withNoProjects' and `withNoFiles_skip` check the same behavior.